.github/CODEOWNERS

```diff
@@ -8,6 +8,9 @@
 # Carbon AI Chat
 /packages/web-components/src/components/chat      @carbon-design-system/carbon-ai-chat-admins @carbon-design-system/developers-system-reviewers
 
+# Carbon UI Shell
+/packages/react/src/components/uishell      @carbon-design-system/carbon-ui-shell-devs
```

Comment:
@alisonjoseph , should we have a prefix to the team name that shows it's in Carbon Labs? Maybe a team for Carbon Labs then have it tucked under there. It would be easier for pruning later on when this workstream is ended.

Reply:
The team is already nested under labs, but I can update the name to be carbon-ui-shell-labs-devs too https://github.com/orgs/carbon-design-system/teams/carbon-ui-shell-devs

Reply:
updated https://github.com/orgs/carbon-design-system/teams/carbon-ui-shell-labs-devs

Reply:
Got it, I should have clicked through to double check first 😁 

Thank you!